stage0/layout.ld

```diff
@@ -15,7 +15,6 @@
 */
 
 HIDDEN(TOP = 4096M);
-HIDDEN(BIOS_SIZE = 512K);
 
```

Comment:
Is it worth adding a comment saying that BIOS_SIZE needs to be provided externally?

Reply:
Done.